src/container.php

```diff
@@ -146,24 +146,22 @@
 $sc->register('app.currency.converter', Goteo\Library\Converter::class);
 
 $sc->register('dispatcher', Symfony\Component\EventDispatcher\EventDispatcher::class)
-  ->addMethodCall('addSubscriber', array(new Reference('app.listener.exception')))
-  ->addMethodCall('addSubscriber', array(new Reference('app.listener.session')))
-  ->addMethodCall('addSubscriber', array(new Reference('app.listener.auth')))
-  ->addMethodCall('addSubscriber', array(new Reference('app.listener.origin')))
-  ->addMethodCall('addSubscriber', array(new Reference('app.listener.project')))
-  ->addMethodCall('addSubscriber', array(new Reference('app.listener.invest')))
-  ->addMethodCall('addSubscriber', array(new Reference('app.listener.poolinvest')))
-  ->addMethodCall('addSubscriber', array(new Reference('app.listener.donateinvest')))
-  ->addMethodCall('addSubscriber', array(new Reference('app.listener.messages')))
-  ->addMethodCall('addSubscriber', array(new Reference('app.listener.project_post')))
-  ->addMethodCall('addSubscriber', array(new Reference('app.listener.blog_post')))
-  ->addMethodCall('addSubscriber', array(new Reference('app.listener.stories')))
-  ->addMethodCall('addSubscriber', array(new Reference('app.listener.channel')))
-  ->addMethodCall('addSubscriber', array(new Reference('console.listener.milestone')))
-  ->addMethodCall('addSubscriber', array(new Reference('console.listener.favourite')))
```

Comment:
@davidbeig @microstudi any good reason why would we want to have console.listener.milestone and console.listener.favourite added both for the EventDispatcher registered as 'dispatcher' and for the 'console_dispatcher' as well?

Could we have it only in the 'console_dispatcher'?

Reply:
Yes, per the ConsoleMilestoneListener, it listens to events defined in ConsoleEvents and in AppEvents. Some are thrown inside a console command, mostly executed using cron, and the others are thrown using the workflow of the application (In this case through and invest)

Reply:
Because the ConsoleMilestoneListener is used both in the web application and the console, events managed are defined here:
https://github.com/GoteoFoundation/goteo/blob/0a5fc684813287e011fcaa77e151d16a73dad002/src/Goteo/Console/EventListener/ConsoleMilestoneListener.php#L150-L157

Maybe it could be done in 2 different classes for clarity with some composition (a trait?) for sharing methods?

Reply:
For now, I'll simply put them back. Let's not delay the fix.